R/z_animint.R

```diff
@@ -200,7 +200,7 @@ parsePlot <- function(meta, plot, plot.name){
   options_list <- getWidthAndHeight(plot$theme)
   options_list <- setUpdateAxes(plot$theme, options_list)
   plot.info$options <- options_list
-
+  plot.info$attributes <- theme_attribute(plot$theme)  
```

Comment:
attributes and theme_attribute are not very informative names.

```suggestion
  plot.info$span <- get_span(plot$theme)
```